src/components/FormGroup/FormGroup.tsx

```diff
@@ -98,6 +98,13 @@ export const FormGroup: React.FC<Props & ElementProps> = ({
         .join(' '),
     [helpMessage, exampleMessage, supplementaryMessage, errorMessages, managedHtmlFor],
   )
+  const actualErrorMessages = useMemo(() => {
+    if (!errorMessages) {
+      return []
+    }
+
+    return Array.isArray(errorMessages) ? errorMessages : [errorMessages]
+  }, [errorMessages])
```

Comment:
statusLabelProps のように `exampleMessages = []` とするのはどうですか?

```suggestion
  const actualErrorMessages = useMemo(
    () => (Array.isArray(errorMessages) ? errorMessages : [errorMessages]),
    [errorMessages],
  )
```

Reply:
@uknmr ↑のロジックでは errorMessagesがfalsy(空文字、nullなど)の場合に問題になります。
nullだと仮定すると

```
const actualErrorMessages = useMemo(
	() => (Array.isArray(null) ? null : [null]),
	[null],
)
```

となり、`actualErrorMessages` は `[null]` という配列になってしまいます。
errorMessagesはReactNodeを型にしているため、nullを受け取れることから、可能性は低いとはいえチェックはしておいたほうが良さそうです。
(利用者が意図していなくてもnull返すコンポーネントを食わせられるので...)

Reply:
手元でも `null` を食べさせるコードを書いて試してたんですが、特に壊れるわけではないので、そこまで担保しなくて良いと思いました。
`statusLabelProps` も同じだし、他コンポーネントでも存在しそう。

Reply:
statusLabelPropsはnullが渡ってくる可能性ないので状況違いますね (Objectなんで...)

Reply:
個人的な意見ですが、この手の壊れずエラーが出ない、気づきにくい表示崩れは原因特定がし辛いし、解決方法も負債にはならないのなら対応してあるほうが良いと思います

Reply:
👍